src/edit/selection.js

```diff
@@ -130,7 +130,7 @@ class SelectionState {
     let anchor = DOMFromPos(this.pm, this.range.anchor)
     let head = DOMFromPos(this.pm, this.range.head)
 
-    let sel = window.getSelection(), range = document.createRange()
+    let sel = this.pm.root.getSelection(), range = document.createRange()
```

Comment:
Can this code be executed when `this.pm.root` is `null`? If so a null check may be worthwhile.